client/my-sites/checkout/checkout-thank-you/premium-plan-details.jsx

```diff
@@ -59,9 +59,16 @@ const PremiumPlanDetails = ( { selectedSite, sitePlans, selectedFeature, purchas
 			/>
 
 			<QuerySiteVouchers siteId={ selectedSite.ID } />
-			<div>
-				<GoogleVoucherDetails selectedSite={ selectedSite } />
-			</div>
+			<PurchaseDetail
+				alt=""
+				id="google-credits"
+				icon={ <img alt="" src="/calypso/images/illustrations/google-adwords.svg" /> }
+				title={ i18n.translate( 'Google Ads credit' ) }
+				description={ i18n.translate(
+					'Use a $100 credit with Google to bring traffic to your most important Posts and Pages.'
+				) }
+				body={ <GoogleVoucherDetails selectedSite={ selectedSite } /> }
+			/>
```

Comment:
We're duplicating information between `GoogleVoucher` and this component. Could we find a way to address that?

Reply:
The reason I didn't do this was because using this component (client/blocks/product-purchase-features-list/google-vouchers.jsx) felt wrong in the checkout thank you.  I would like to dry it up but I think that refactor is out of scope for this PR.